core/vm/ipgraph.go

```diff
@@ -10,10 +10,17 @@ import (
 	"github.com/ethereum/go-ethereum/log"
 )
 
+const (
+	ipGraphWriteGas        = 100
```

Comment:
Does the write gas vary based on ip graph size?

Reply:
per current ipgraph design, the cost of write operation to ipgraph  is constant, would not change with graph size.

Reply:
It would good to have our own take on the pricing before bringing external consultants to review.

Reply:
Good point, we can further fine tune the gas. Have ticket to address this effort: https://github.com/storyprotocol/protocol-core-v1/issues/231